src/rgw/driver/rados/rgw_rest_pubsub.h

```diff
@@ -25,11 +25,13 @@ class RGWHandler_REST_PSNotifs_S3 : public RGWHandler_REST_S3 {
 // AWS compliant topics handler factory
 class RGWHandler_REST_PSTopic_AWS : public RGWHandler_REST {
   const rgw::auth::StrategyRegistry& auth_registry;
+  bufferlist bl_post_body;
```

Comment:
why do we need to actually store the body here?
can't we use a const& data member?

Reply:
the bufferlist is on the stack of [RGWRESTMgr_S3::get_handler()](https://github.com/ceph/ceph/blob/wip-rgw-meta-topic/src/rgw/rgw_rest_s3.cc#L5200-L5213) and goes out of scope right after

Reply:
so, if we don't care about the original bufferlist, why not use `std::move()`?

Reply:
thanks, updated